### PR TITLE
fixing french and adding in locales

### PR DESCRIPTION
**Pull request by sarahboyd**
Fixing a couple plural terms so they don't break in the UI

@imcdonald-d2l also noticed a bunch of plural terms not properly translated, I'll submit them to the translation team to get updated

### CODE REVIEW

File: lang/pt.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
    "moduleListExpanded" : "Lista de módulos expandida",
    "newContentAvailable" : "Há um novo conteúdo disponível.",
    "niceWork" : "Bom trabalho até agora!",
-   "noNeedToFinish" : "Se você não precisar concluir {count, plural, one {it} other {they}}, não há problema.",
+   "noNeedToFinish" : "Se você não precisar concluir {count, plural, one {# it} other {# they}}, não há problema.",
```
Comment (imcdonald-d2l) [inline lang/pt.json:33]: is this supposed to be `them`? not `they`?

Comment (imcdonald-d2l) [reply lang/pt.json:33]: Same in `pt-PT`

Comment (sarahboyd) [reply lang/pt.json:33]: lollllllllllllllllllllll

File: locales.json [hunk 2]
```diff
@@ -984,7 +1040,7 @@
 		"moduleListExpanded": "Lista de módulos expandida",
 		"newContentAvailable": "Há um novo conteúdo disponível!",
 		"niceWork": "Bom trabalho até agora!",
-		"noNeedToFinish": "Se você não precisar concluir {count, plural, one {it} other {they}}, não há problema.",
+		"noNeedToFinish": "Se você não precisar concluir {count, plural, one {# it} other {# they}}, não há problema.",
```
Comment (imcdonald-d2l) [inline locales.json:1043]: sneaky sneaky... another spelling issue.

File: locales.json [hunk 3]
```diff
@@ -1038,10 +1094,10 @@
 		"module": "Módulo",
 		"moduleListCollapsed": "Lista de módulos recolhida",
 		"moduleListExpanded": "Lista de módulos expandida",
-		"newContentAvailable": "Há um novo conteúdo disponível!",
+		"newContentAvailable": "Há um novo conteúdo disponível.",
 		"niceWork": "Bom trabalho até agora!",
-		"noNeedToFinish": "Se você não precisar concluir {count, plural, one {it} other {they}}, não há problema.",
-		"openDocReader": "Open DocReader",
+		"noNeedToFinish": "Se você não precisar concluir {count, plural, one {# it} other {# they}}, não há problema.",
```
Comment (imcdonald-d2l) [inline locales.json:1099]: spelling issue again

**Pull request merged**